cfme/tests/containers/test_containers_properties.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.containers.pod import Pod, list_tbl as list_tbl_pods
+from cfme.containers.route import Route, list_tbl as list_tbl_routes
+from cfme.containers.project import Project, list_tbl as list_tbl_projects
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope="function")
+
+# CMP-9911 # CMP-9877 # CMP-9867
+
+
+@pytest.mark.parametrize('rel',
+                         ['name',
+                          'phase',
+                          'creation_timestamp',
+                          'resource_version',
+                          'restart_policy',
+                          'dns_policy',
+                          'ip_address'
+                          ])
+def test_pods_properties_rel(provider, rel):
+    """ This module verifies data integrity in the Properties table for:
+        pods, routes and projects
+    """
+    sel.force_navigate('containers_pods')
+    ui_pods = [r.name.text for r in list_tbl_pods.rows()]
```

Comment:
I see what you mean, PRT cannot find the screenshots because I am using obj.summary instead of obj.get_detail. I'll move this into Pod class right now.